the pull-metrics-server-test-e2e-ha test fails frequently recently, and the  reasons need to be analysed

<!-- 
STOP -- PLEASE READ!

If you're looking for help, check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
Have questions? First please read [Frequently Asked Questions](https://github.com/kubernetes-sigs/metrics-server/blob/master/FAQ.md)
Encountered a problem? First please read [Known Issues](https://github.com/kubernetes-sigs/metrics-server/blob/master/KNOWN_ISSUES.md)
You can also post your question on the [#sig-instrumentation](https://kubernetes.slack.com/messages/sig-instrumentation) channel of [Kubernetes Slack](http://slack.k8s.io/) or the [Discuss Kubernetes](https://discuss.kubernetes.io/) forum.
If the matter is security related, please disclose it privately via https://kubernetes.io/security/.

Please use template below and provide as much info as possible.
Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->

**What happened**:
the `pull-metrics-server-test-e2e-ha`  e2e tests of metrics-server  fails frequently recently, and the  reasons need to be analysed.
such as :
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_metrics-server/969/pull-metrics-server-test-e2e-ha/1498266418871799808
reason of failure:
```
[Fail] MetricsServer [It] exposes prometheus metrics 
/home/prow/go/src/sigs.k8s.io/metrics-server/test/e2e_test.go:309
Ran 9 of 9 Specs in 44.442 seconds
FAIL! -- 8 Passed | 1 Failed | 0 Pending | 0 Skipped
--- FAIL: TestMetricsServer (44.45s)
FAIL
FAIL	command-line-arguments	44.461s
FAIL
```
**What you expected to happen**:
fix the e2e  ha tests.
**Anything else we need to know?**:

**Environment**:
- Kubernetes distribution (GKE, EKS, Kubeadm, the hard way, etc.):
- Container Network Setup (flannel, calico, etc.):
- Kubernetes version (use `kubectl version`):

- Metrics Server manifest

<details>
  <summary>spoiler for Metrics Server manifest:</summary>

  <!--- INSERT manifest HERE --->

</details>

- Kubelet config:

<details>
  <summary>spoiler for Kubelet config:</summary>

  <!--- INSERT kubelet config HERE --->

</details>

- Metrics server logs:

<details>
  <summary>spoiler for Metrics Server logs:</summary>

  <!--- INSERT logs HERE --->

</details>

- Status of Metrics API:
<details>
  <summary>spolier for Status of Metrics API:</summary>

  ```
  kubectl describe apiservice v1beta1.metrics.k8s.io
  ```

  <!--- INSERT results of command above --->

</details>

<!-- DO NOT EDIT BELOW THIS LINE -->
/kind bug

--- a/test/e2e_test.go
+++ b/test/e2e_test.go
@@ -223,8 +223,11 @@ livez check passed
 		}
 	})
 	It("exposes prometheus metrics", func() {
+
 		msPods := mustGetMetricsServerPods(client)
 		for _, pod := range msPods {
+			_, err := proxyRequestToPod(restConfig, pod.Namespace, pod.Name, "https", 4443, "/apis/metrics.k8s.io/v1beta1/")
+			Expect(err).NotTo(HaveOccurred(), "Failed to get Metrics Server /apis/metrics.k8s.io/v1beta1/ endpoint")
 			resp, err := proxyRequestToPod(restConfig, pod.Namespace, pod.Name, "https", 4443, "/metrics")
 			Expect(err).NotTo(HaveOccurred(), "Failed to get Metrics Server /metrics endpoint")
 			metrics, err := parseMetricNames(resp)

--- a/test/e2e_test.go
+++ b/test/e2e_test.go
@@ -223,9 +223,9 @@ livez check passed
 		}
 	})
 	It("exposes prometheus metrics", func() {
-
 		msPods := mustGetMetricsServerPods(client)
 		for _, pod := range msPods {
+			// access /apis/metrics.k8s.io/v1beta1/ for each pod to ensures that every MS instance get an requests so they expose all apiserver metrics.
 			_, err := proxyRequestToPod(restConfig, pod.Namespace, pod.Name, "https", 4443, "/apis/metrics.k8s.io/v1beta1/")
 			Expect(err).NotTo(HaveOccurred(), "Failed to get Metrics Server /apis/metrics.k8s.io/v1beta1/ endpoint")
 			resp, err := proxyRequestToPod(restConfig, pod.Namespace, pod.Name, "https", 4443, "/metrics")

